parm/config/config.fcst

```diff
@@ -103,6 +103,9 @@ export ICEPETS="80"
 export WAVPETS="160"
 export CHMPETS="1700"
 
+[[ "$CDUMP" = "gfs" ]] && ntasks_fcst=$npe_fcst_gfs || ntasks_fcst=$npe_fcst
```

Comment:
Please expand this out into a proper if/then/else block. I'm going to be removing all of this sort of short-circuit soon.

Reply:
Can I ask why?
List comprehension is a standard technique, succinct and clear.

Reply:
What does "short-circuit" means here? I mean they are equivalent to if/then/else logically.

Reply:
This is addressed.